english/candyland/obamacare/hillary/hillary-appears.md

```diff
@@ -0,0 +1,6 @@
+After passing out from a lack of insulin, you realize that someone is standing over you. It's Hillary Clinton, holding out a cup of grape juice befitted with a straw.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.